### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by lisa-fehr**
Closes: #2702 

Altered translation with colon. Wasn't parsing correctly, might be possible to escape the colon.
Adding missing translations.

`CLOSE` may still show up as a missing translation when adding a Script Task and expanding the Script Configurator Editor. This will eventually go away once this pr has a release # https://github.com/ProcessMaker/screen-builder/pull/491

### CODE REVIEW

File: resources/lang/en.json [hunk 1]
```diff
@@ -127,6 +128,7 @@
   "Client ID": "Client ID",
   "Client Secret": "Client Secret",
   "Close": "Close",
+  "CLOSE": "CLOSE",
```
Comment (chipit24) [inline resources/lang/en.json]: I don't think we should have different translation for an all-caps version of the word "close". I think the translation above this one ("Close") should be used, and a style of `text-transform: capitalize;` should be applied to it. There is a bootstrap helper for this too: https://getbootstrap.com/docs/4.0/utilities/text/#text-transform.

```
<p class="text-lowercase">Lowercased text.</p>
<p class="text-uppercase">Uppercased text.</p>
<p class="text-capitalize">CapiTaliZed text.</p>
```

File: resources/lang/en.json [hunk 2]
```diff
@@ -363,13 +366,16 @@
   "Group Permissions Updated Successfully": "Group Permissions Updated Successfully",
   "Group Permissions": "Group Permissions",
   "Group": "Group",
+  "group": "group",
```
Comment (chipit24) [inline resources/lang/en.json]: Same thing here regarding capitalization. I think capitalization should be handled with CSS and whether certain words are capitalized or not can be controlled with the bootstrap text helper classes.

File: resources/lang/en.json (hunk 1 above)
Comment (lisa-fehr) [reply resources/lang/en.json]: I thought I found `CLOSE` in screen builder, but when I change it and link it into processmaker it doesn't change.

File: resources/lang/en.json (hunk 2 above)
Comment (lisa-fehr) [reply resources/lang/en.json]: When searching, I have trouble finding where `Group` and `group` occur. Some are headers and some are select options. Switching all instances of `Group` might be tricky for me. I'm not sure when/where the `group` translation was introduced.

Comment (velkymx) [reply resources/lang/en.json]: This is fine for now. We'll circle back if becomes an issue.

**Pull request merged**